community/rgbds/depends

```diff
@@ -0,0 +1,4 @@
+bison make
+pkgconf make
+libpng make
```

Comment:
Should this be a `make` dependency? The software doesn't dynamically link to this / need it at runtime?

Reply:
Hmm, I haven’t used much of the particular program that pulls in the libpng dependency. The RGBDS website only lists libpng’s development headers as a dependency, rather than the whole library. Let me test the relevant program tomorrow on a machine without libpng (and only the headers) to see if it works.

Reply:
You can also check by running `ldd <file>` for each file in the package.